docs/website/content/v0.3/en/configuration/v1alpha1.md

```diff
@@ -0,0 +1,276 @@
+```
+
+nolint: lll
+
+## Reference
+
+### type
```

Comment:
i feel like these keys need to be fully qualified `machine.type` unless we split things up into `machine` config reference and `cluster` config reference docs.

Reply:
The best I could come up with is to have all options related to a struct under a common header. Checkout the new web preview. The Golang AST stuff is pretty hairy. Maybe something we can look at later, but with the sections it is actually pretty nice IMHO. We don't need to have a fully qualified since the entire section is about the specific struct.